repos/system_upgrade/el7toel8/actors/removeoldlog/actor.py

```diff
@@ -0,0 +1,18 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.library import remove_log
+from leapp.tags import IPUWorkflowTag, FactsPhaseTag
+
+
+class RemoveOldLog(Actor):
+    """
+    Removes the old log from the previous Leapp run
```

Comment:
1st line should be a summary and details with one empty line in between should follow.